### PR TITLE
Lots of fixes

### PR DESCRIPTION
**Pull request by CJEnright**
Updates to application are:
* Add phone number
* Add Where did you hear about BM
* Add Is First Hackathon checkbox
* Remove LinkedIn

### CODE REVIEW

File: templates/web/components/bmvii_navbar.tmpl [hunk 1]
```diff
@@ -22,10 +22,10 @@
 		</div>
 
 		<div class="navbar-end">
-			<a class="navbar-item">
+			<a class="navbar-item";href="https://www.instagram.com/boilermake/">
```
Comment (romulus3799) [inline templates/web/components/bmvii_navbar.tmpl]:
```suggestion
			<a class="navbar-item" href="https://www.instagram.com/boilermake/">
```

File: templates/web/pages/apply.tmpl [hunk 2]
```diff
@@ -219,34 +234,86 @@
...
+						{{ else }}
+						<input type="checkbox" name="is-first-hackathon" class="checkbox">
+						{{ end }}
+						<label for="is-first-hackathon" class="checkbox">This is my first hackathon</label>
+					</div>
+
 					<div class="field is-horizontal">
 						<div class="field-label is-normal">
-							<label for="linkedin" class="label">LinkedIn</label>
+							<label for="race" class="label">Where did you hear about BoilerMake?</label>
```
Comment (romulus3799) [inline templates/web/pages/apply.tmpl]: Uhhhh race? Shouldn't this be referrer?

Comment (CJEnright) [reply templates/web/pages/apply.tmpl]: And that's why we do code reviews.  Good catch

File: templates/web/pages/sponsors.tmpl [hunk 3]
```diff
@@ -30,7 +30,7 @@
 		<div class="columns is-centered content">
 			<div class="column is-8">
 				<div class="columns is-centered content">
-					<div class="column is-8">
+					<div class="column is-8 sponsors__legacy-header">
```
Comment (CJEnright) [inline templates/web/pages/sponsors.tmpl]: Bulma has a class built in for this
```suggestion
					<div class="column is-8 has-text-centered">
```

File: web/src/styles/index.scss [hunk 4]
```diff
@@ -159,3 +166,7 @@ $dark: #231F20;
 .bmvii-section-header {
 	padding-top: 2rem;
 }
+
+.sponsors__legacy-header {
+	text-align: center;
+}
```
Comment (CJEnright) [inline web/src/styles/index.scss]: Can delete if you make change in `sponsors.tmpl`

**Pull request merged**